.github/ISSUE_TEMPLATE/05-community-toolkit-docs-request.yml

```
@@ -0,0 +1,71 @@
+name: "Community Toolkit article request"
+description: Request a new article on a topic or concept related to the [Aspire Community Toolkit](https://github.com/CommunityToolkit/Aspire) that you can't find in our current docs.
+title: "[Community Toolkit New article]: "
+labels: ["doc-idea", "community-toolkit"]
+assignees: ["Alirexaa", "aaronpowell"]
+body:
+  - type: markdown
+    attributes:
+      value: "## Request a new article"
+  - type: markdown
+    attributes:
+      value: "Describe the proposed new article for us."
+  - type: input
+    id: topic
+    attributes:
+      label: Proposed topic or title
+      description: Write a short title or description of the topic
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: >
```

Nit: style preference, noticed in other places here in the template.

```suggestion
      value: |
```

---

I believe this is just copied from https://[raw.githubusercontent.com/dotnet/docs-aspire/refs/heads/main/.github/ISSUE_TEMPLATE/02-docs-request.yml](https://raw.githubusercontent.com/dotnet/docs-aspire/refs/heads/main/.github/ISSUE_TEMPLATE/02-docs-request.yml) which uses `>`.